CHANGELOG

```diff
@@ -1,3 +1,7 @@
+# 0.2.4
+
+* Add the `--health` option to `kup list` showing the lag behind master for each dependency
```

Comment:
```suggestion
* Add the `--health` option to `kup list` showing the lag behind the default branch for each dependency
```

Reply:
Also, consider the name `--status` (what this command does is similar to `git status`).